conanfile.py

```diff
@@ -3,7 +3,7 @@
 
 class PolarsConan(ConanFile):
     name = "Polars"
-    version = "0.1.0"
+    version = "0.1.1"
```

Comment:
I think we discussed making this a minor version bump, not patch?

Reply:
Gah. My bad. Amending.